test/common/upstream/least_request_load_balancer_fuzz_test.cc

```diff
@@ -45,7 +45,11 @@ DEFINE_PROTO_FUZZER(const test::common::upstream::LeastRequestLoadBalancerTestCa
     ENVOY_LOG_MISC(debug, "ProtoValidationException: {}", e.what());
     return;
   }
-
+  if (input.least_request_lb_config().has_choice_count() &&
+      input.least_request_lb_config().choice_count().value() > 10000) {
```

Comment:
seems reasonable, I think the num hosts in a locality is <= 60k. Why 10k vs something smaller? Seems like we'd be spending a lot of time in https://github.com/envoyproxy/envoy/blob/b51d93bbd2e0fafaad9d550f7afaa219f493d637/source/common/upstream/load_balancer_impl.cc#L1029-L1045 which doesn't seem too interesting

Reply:
I agree we can constraint it even further. IIUC any value above 100 will make sense, and we just want to make sure the fuzzer doesn't spend too much time on functions such as these.